src/components/text-input/TextInput.js

```diff
@@ -46,29 +45,28 @@ const TextInput = ({
     labelTooltip,
     ...rest
 }: Props) => {
-    const classes = classNames(className, 'text-input-container', {
-        'show-error': !!error,
+    const classes = classNames(className, 'bdl-TextInput-container', {
+        'bdl-hasError': !!error,
```

Comment:
No TextInput- for this as this is a state that other places can re-use. Downside of this is that people should only have it `&`ed and never styled on its own.